discord/sinks/core.py

```diff
@@ -118,20 +118,22 @@ def __init__(self, data, client):
 class AudioData:
     """Handles data that's been completely decrypted and decoded and is ready to be saved to file.
 
-    .. versionadded:: 2.1
-
-    Raises
-    ------
-    ClientException
-        The AudioData is already finished writing,
-        The AudioData is still writing
+    .. versionadded:: 2.0
     """
 
     def __init__(self, file):
         self.file = file
         self.finished = False
 
     def write(self, data):
+        """
```

Comment:
may i suggest using the [pep 257 convention](https://peps.python.org/pep-0257) of docstrings
```py
"""short

longer
description
"""
```